app/controllers/application_controller.rb

```diff
@@ -1,2 +1,36 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+ protected
+
+ def configure_permitted_parameters
+   devise_parameter_sanitizer.permit(:sign_up, keys: [:nombre, :image])
+   devise_parameter_sanitizer.permit(:account_update, keys: [:nombre, :image])
```

Comment:
you're naming variables in spanish, remember that your code must be consistent. So, all variables/functions should be named in English

Reply:
Sorry I know it u.u)/. I tried to change this part but a can't :cactus: 
I'm going to try to change with calm ...